widgets/BigQueryTable-batchsource.json

```diff
@@ -21,6 +21,13 @@
           "name": "project",
           "widget-attributes" : {
             "default": "auto-detect"
+          },
+          "plugin-function": {
```

Comment:
does this get ignored by earlier versions of the UI?

Reply:
The plugin function as a concept got removed in the backend. I don't believe we changed the spec to reflect that. In plugins like [Database source](https://github.com/cdapio/hydrator-plugins/blob/develop/database-plugins/widgets/Database-batchsource.json#L47) it still exists.